packages/element-library/src/elementTypes.js

```diff
@@ -18,6 +18,7 @@
  * External dependencies
  */
 import { __ } from '@googleforcreators/i18n';
+import { ELEMENT_TYPES } from '@googleforcreators/elements';
```

Comment:
I am a bit hesitant to use `ELEMENT_TYPES` more, because originally the idea behind this abstraction and decoupling effort was that any platform/editor using our packages could register their own additional element types. Which is why `elements` and `element-library` are separate packages.

Unfortunately we never 100% followed through with that abstraction, so there's still too many interdependencies for this to work reliably. So I suppose this is OK for now.

Reply:
I tend to agree. I don't think this is the right direction to go. I don't think this PR is warranted at all in terms of using `ELEMENT_TYPES` everywhere. Especially not in tests. The part about using elementIs where-ever possible is definitely a good idea, but using `ELEMENT_TYPES` over plain strings doesn't seem to do anything useful to me.